From 87d2bfeef70bc9d017472c38c663933c911b9091 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Mon, 29 Oct 2018 18:54:05 +0200
Subject: [PATCH 715/767] arm64: dts: lx2160a-rdb: Add Aquantia PHY nodes

DPMAC3 and DPMAC4 are connected to 10G Aquantia PHYs

Signed-off-by: Catalin Neacsu <valentin-catalin.neacsu@nxp.com>
Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 .../boot/dts/freescale/fsl-lx2160a-rdb.dts    | 22 +++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
index c3bb4dd43edf..b6917d8e7377 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
@@ -159,6 +159,18 @@
 		reg = <0x2>;
 		/* Poll mode - no "interrupts" property defined */
 	};
+	aquantia_phy1: ethernet-phy@4 {
+		/* AQR107 PHY - "compatible" property not strictly needed */
+		compatible = "ethernet-phy-ieee802.3-c45";
+		reg = <0x4>;
+		/* Poll mode - no "interrupts" property defined */
+	};
+	aquantia_phy2: ethernet-phy@5 {
+		/* AQR107 PHY - "compatible" property not strictly needed */
+		compatible = "ethernet-phy-ieee802.3-c45";
+		reg = <0x5>;
+		/* Poll mode - no "interrupts" property defined */
+	};
 };
 
 &emdio2 {
@@ -170,6 +182,16 @@
 	};
 };
 
+&dpmac3 {
+	phy-handle = <&aquantia_phy1>;
+	phy-connection-type = "xgmii";
+};
+
+&dpmac4 {
+	phy-handle = <&aquantia_phy2>;
+	phy-connection-type = "xgmii";
+};
+
 &dpmac5 {
 	phy-handle = <&inphi_phy>;
 };
-- 
2.17.0

